synapse/handlers/auth.py

```diff
@@ -856,15 +870,17 @@ def add_threepid(self, user_id, medium, address, validated_at):
         )
 
     @defer.inlineCallbacks
-    def delete_threepid(self, user_id, medium, address, id_server=None):
+    def delete_threepid(
+        self, user_id: str, medium: str, address: str, id_server: Optional[str] = None
+    ):
         """Attempts to unbind the 3pid on the identity servers and deletes it
         from the local database.
 
         Args:
-            user_id (str)
-            medium (str)
-            address (str)
-            id_server (str|None): Use the given identity server when unbinding
+            user_id
+            medium
+            address
```

Comment:
I think these can be removed now, since they were mainly here to indicate types (which IDEs would then be able to parse).

Reply:
Ideally we'd fill out the docstrings, but just having them floating here is a little odd, yeah. 👍